fe1-web/src/features/rollCall/screens/RollCallScanner.tsx

```diff
@@ -111,9 +111,13 @@ const RollCallOpened = () => {
 
   const addAttendeePopToken = useCallback(
     (popToken: string) => {
+      // if popToken is not in base64 format, do not add it
+      if (!popToken.match(/^[a-zA-Z0-9_-]+={0,3}$/) || popToken.length % 4 !== 0) {
+        throw new Error('Invalid popToken format');
+      }
+
```

Comment:
I think it is better if this is moved to at least another function or the `ScannablePopToken.fromJson` which is used line 144